### DISCUSSION TITLE
Dagster UI not loading after switching to Postgres run storage

### DISCUSSION THREAD

**Discussion by EvanZ** [dagster-io/dagster#23939] [Q&A]
I was running dagster UI with sqllite storage and having no issues. I wanted to switch to postgres and have set that up using `dagster.yaml`. Dagster created all the necessary tables on the postgres server (running 11.22) but when I launch the UI with `dagster dev` I just see a blank Chrome tab with a spinner. Any ideas what could be at issue here?

**Comment by garethbrickman** [comment]
If you open Chrome's developer tools is there anything being logged in the console, or any erroring network requests?

**Reply by EvanZ** [reply]
Oddly it resolved on its own the next morning. Not sure what happened if it got into a weird state or something.
